src/mosaic_cache_invalidation_job.mjs

```diff
@@ -117,12 +117,17 @@ async function invalidateMosaicCache() {
     // all cached mosaic tiles that contain this image need to be invalidated
     // because the image itself was deleted.
     if (!image) {
-      const metadataBuffer = await cacheGet(metadataCacheKey);
-
-      if (!metadataBuffer || !metadataBuffer.length) continue; // skip empty metadata jsons
-
-      const metadata = JSON.parse(metadataBuffer.toString());
-      const { bounds, maxzoom } = metadata;
+      let bounds, maxzoom;
+      try {
+        const metadataBuffer = await cacheGet(metadataCacheKey);
+        if (!metadataBuffer || !metadataBuffer.length) continue;
+        const metadata = JSON.parse(metadataBuffer.toString());
+        if (!metadata) continue;
+        bounds = metadata.bounds;
+        maxzoom = metadata.maxzoom;
+      } catch (error) {
+        continue; // skip empty or broken metadata jsons
```

Comment:
shouldn't we know some info if error appeared in case if the metadata is present? catching error and not pasting it anywhere seems to be strange

Reply:
added warning